### ISSUE TITLE
Step 4 is unclear

### ISSUE THREAD

**Issue by andreaferretti** [personalized-restoration/personalized-restoration#3]
Hello, I am trying to follow the instructions in the README, but I am finding step 4 unclear. Namely, it reads

> First, you need to create a file list of training set and validation set. Please follow guidelines from the DiffBIR repository, under the heading ``Data Preparation: 1.Generate file list of training set and validation set'' in their repository.

but it does not say what should be used as training or validation set (I don't think a validation set is even needed, it can just be skipped). From the paper, I understand that one should use a general dataset of faces, but looking at the dataset in `codeformer.py`, I see that the prompt `a photo of a sks man` is hardcoded, whereas I would have expected `a photo of a man`, or even `a photo of a person` when finetuning the ControlNet on a general photo. Is this correct?

**Comment by personalized-restoration** [comment]
Hi! Thanks for your question. You can use the provided example images in the trial_data directory as train set images (as you mentioned, no validation set images). In terms of the prompt, we use the 'sks' identifier that relates with the first stage of personalization, using DreamBooth: identity information is tuned within the 'sks' token. If you would like to use a different gender prior (for example, if you use images of a female person), you can change the prompt accordingly. Hope this helps!

**Comment by andreaferretti** [comment]
Thank you for the prompt response. What I don't understand is the following: during the training of Dreambooth in the first phase we want to associate the identity with a rare token; sks is a common choice for that.

But then when training the second phase (model pivoting) we are using different identities: the aim here is to realign the IRControlNet. Why use sks in this context?

**Comment by andreaferretti** [comment]
In addition, I am not sure about this comment

> You can use the provided example images in the trial_data directory as train set images

The provided example images are all of the same identity. But the model based pivoting step is performed for different identities

**Comment by andreaferretti** [comment]
My bad, by training on the same identity this actually works. I am slightly confused, as the paper seemed to imply something different (with the step Identity-agnostic guidance preservation)
